Client/Application/AppDelegate.swift

```diff
@@ -111,7 +111,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
 
         // Start intialzing the Nimbus SDK. This should be done after Glean
         // has been started.
-        initializeExperiments()
+        let defaults = UserDefaults.standard
+        let nimbusFirstRun = "NimbusFirstRun"
+        let isFirstRun = defaults.bool(forKey: nimbusFirstRun) != false
+        defaults.set(false, forKey: nimbusFirstRun)
+        initializeExperiments(customTargetingAttributes: ["isFirstRun": String(isFirstRun)], isFirstRun: isFirstRun)
```

Comment:
Passing in both as a dictionary entry and as a boolean for the same parameter seems redundant...

Prefer string interpolation over constructor for type conversion into String:
```suggestion
        initializeExperiments(customTargetingAttributes: ["isFirstRun": "\(isFirstRun)"], isFirstRun: isFirstRun)
```